themegrill/spacious
# Added font-display property for faster load time.

Changes proposed in this Pull Request

Adds 'font-display:swap' CSS property for fonts in the site. This ensures fonts remains visible on page reload and also ensures better performance.

Type of change

[ ] Code Refactor

[X] Bug fix (non-breaking change which fixes an issue)

[ ] New feature (non-breaking change which adds functionality)

[ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

[ ] This change requires a documentation update

How to test the changes in this Pull Request:

One can test by using the lighthouse tool available in the chrome developers tool.

While testing it there should not be any warning message which says 'Ensure text remains visible during page reload'

Checklist:

[x] My code follows WordPress' coding standards

[x] I've checked to ensure there are no other open Pull Requests for the same update/change?

[x] I have performed a self-review of my own code

[x] I have commented on my code, particularly in hard-to-understand areas

[ ] I have made corresponding changes to the documentation

[x] My changes generate no new warnings

[x] I have made perform a test that proves my fix is effective or that my feature works

Did you test this issue fix on all browsers?

[x] Chrome

[x] Firefox

[ ] Edge

[ ] opera
